span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/SpanFilter.java

```diff
@@ -88,14 +119,23 @@ public SpanFilter(Config config) {
    * the span should be dropped, false otherwise.
    */
   public boolean shouldDropSpan(
-      JaegerSpanInternalModel.Span span, Map<String, JaegerSpanInternalModel.KeyValue> tags) {
+      JaegerSpanInternalModel.Span span,
+      Map<String, JaegerSpanInternalModel.KeyValue> tags,
+      Map<String, JaegerSpanInternalModel.KeyValue> processTags) {
     if (anyCriteriaMatch(tags, spanDropCriterion)) {
```

Comment:
this should be removed now?

Reply:
Planning to do as follow up PR once this is deployed.

Reply:
Done